test/integration/create-next-app/templates-pages.test.ts

```diff
@@ -256,6 +256,11 @@ describe('create-next-app templates', () => {
         ],
         {
           cwd,
+          // @ts-expect-error process.env.NODE_ENV is not always avaliable in create-next-app
+          env: {
+            // Use NO_COLOR=1 to test picocolors
```

Comment:
```suggestion
            // Since picocolors assumes `CI=1` means colors are supported
            // this test will fail in GitHub Actions unless we disable color here.
            // See https://github.com/alexeyraspopov/picocolors/blob/6b43e8e83bcfe69ad1391a2bb07239bf11a13bc4/picocolors.js#L9
```